test/test_restapi.py

```diff
@@ -795,6 +795,28 @@ def test_patch_update_routes_with_optional_args(self, setup_restapi_client):
         j = json.loads(r.text)
         assert sorted(j) == sorted(routes)
 
+        # Endpoint Monitor optional arg
+        route['vnid'] = 5000
+        route['nexthop_monitor'] = '100.3.152.32,200.3.152.32'
+        route['cmd'] = 'add'
+        r = restapi_client.patch_config_vrouter_vrf_id_routes("vnet-guid-1", [route])
+        assert r.status_code == 204
+        route_table = db.hgetall(ROUTE_TUN_TB + ':' + VNET_NAME_PREF +str(1)+':'+route['ip_prefix'])
+        assert route_table == {b'endpoint' : route['nexthop'],
+                                       b'vni': str(route['vnid']), 
+                                       b'mac_address' : route['mac_address'],
+                                       b'nexthop_monitor': route['nexthop_monitor']
```

Comment:
in the app_db, it must be '`endpoint_monitor`' as swss handles only endpoint_monitor